src/FrequencyMapAgent.cpp

```diff
@@ -441,5 +485,22 @@ namespace geopm
         m_core_freq_max = m_platform_io.read_signal("CPU_FREQUENCY_MAX_AVAIL", GEOPM_DOMAIN_BOARD, 0);
         m_uncore_init_min = m_platform_io.read_signal("CPU_UNCORE_FREQUENCY_MIN_CONTROL", GEOPM_DOMAIN_BOARD, 0);
         m_uncore_init_max = m_platform_io.read_signal("CPU_UNCORE_FREQUENCY_MAX_CONTROL", GEOPM_DOMAIN_BOARD, 0);
+
+        
+        m_gpu_freq_ctl_domain_type = m_platform_io.control_domain_type("GPU_CORE_FREQUENCY_MAX_CONTROL");
+        if (m_gpu_freq_ctl_domain_type == GEOPM_DOMAIN_INVALID) {
+            m_num_gpu_freq_ctl_domain = 0;
+        }
```

Comment:
What is the purpose of proceeding in the case of an invalid GEOPM domain for "GPU_CORE_FREQUENCY_MAX_CONTROL"? The reason for asking is because I believe ``control_domain_type`` will throw in the case it's unable to determine the domain type for a control.

Reply:
I assumed that if GPUs are not supported on the system, we would get a GEOPM_DOMAIN_INVALID but still want to operate without throwing an error.